flow/templates/print_status.sh

```diff
@@ -0,0 +1,55 @@
+{# The following variables are available to all scripts. #}
+{% set format_progress_bar = ("%%-%ss  %%-%ss" | format(label_length,bar_length,)) %}
+{% if detailed %}
+{% set alias = scheduler_status_code %}
+{% set operation_length = [operation_length,5] | max %}
+{% set format_job_id = ("%%-%ss  " | format(id_length,)) %}
+{% set format_operation = ("%%-%ss %%3s  " | format(operation_length,)) %}
+{% set format_operation_title = ("%%-%ss  " | format(operation_length+4,)) %}
+{% set format_label = ("%%-%ss" | format(total_label_length,)) %}
+{% set ns = namespace(dash='', format_parameters='') %}
+{% if parameters %}
+{% for para in parameters %}
+{% set ns.dash = ns.dash + ("%s  " | format('-' * parameters_length[loop.index-1])) %}
+{% set ns.format_parameters = ns.format_parameters + ("%%-%ss  " | format(parameters_length[loop.index-1],)) %}
+{% endfor %}
+{% set para_head = ns.format_parameters | format(*parameters,) %}
+{% endif %}
+{% set format_head = format_job_id + format_operation_title + '%s' + format_label %}
+{% set format_row = format_job_id + format_operation + '%s' + format_label %}
+{% endif %}
+{% block body %}
+{% if overview %}
+# Overview:
+Total # of jobs: {{'%s' |format(jobs_count)}}
```

Comment:
I think it would be more natural to use the `len` filter here: `{{ jobs | len %}}`.

That should work, right?